docs/reference/errors-and-warnings/NU3008.md

```diff
@@ -12,17 +12,14 @@ f1_keywords:
 
 # NuGet Error NU3008
 
-<pre>Package 'SamplePackage v1.0.0' from source 'https://contoso.com/index.json': The package integrity check failed.</pre>
+<pre>Package 'SamplePackage v1.0.0' from source 'https://contoso.com/index.json': The package integrity check failed. The package has changed since it was signed. Try clearing the local cache and run nuget operation again.</pre>
```

Comment:
Is this actually implemented? https://github.com/NuGet/NuGet.Client/search?q=Try+clearing+the+local+cache+and+run+nuget+operation+again

What does the cache refer to here? there's some ambiguity in using cache only.

Reply:
> Is this actually implemented? 

This change is in review https://github.com/NuGet/NuGet.Client/pull/3992

Reply:
Updated the text in the doc as per the discussion https://github.com/NuGet/NuGet.Client/pull/3992#discussion_r611970910